spec/helper.js

```diff
@@ -61,6 +61,9 @@ if (process.env.PARSE_SERVER_TEST_DB === 'postgres') {
   databaseAdapter = new MongoStorageAdapter({
     uri: mongoURI,
     collectionPrefix: 'test_',
+    mongoOptions: {
+      enableSchemaHooks: true,
+    }
```

Comment:
I believe this is setting the option for all tests. Are you sure this is what you want; or do you want to set this only for a specific test that focuses on this option?